docs/components/table.md

```diff
@@ -629,7 +629,7 @@ You may also adjust the border-radius via `--table-border-radius` variable.
 
 ```css
 :root {
-  --table-border: 1px solid var(--c-divider-2);
+  --table-border: 1px solid var(--c-gutter);
```

Comment:
Ah sorry this needs to be `--c-divider`. This one is for outer border for whole table (for STable .box), and this one should not be gutter.